src/analysis/normed_space/basic.lean

```diff
@@ -84,6 +84,15 @@ calc ∥-g∥ = ∥0 - g∥ : by simp
       ... = ∥g - 0∥ : (dist_eq_norm g 0)
       ... = ∥g∥ : by simp
 
+lemma norm_reverse_triangle' {a b : α} : ∥a∥ - ∥b∥ ≤ ∥a - b∥ :=
```

Comment:
Why are these `a` and `b` implicit? They cannot be inferred from any other argument

Reply:
Good point, changed it :)